### PR TITLE
Write test for issue 15

### PR DESCRIPTION
**Pull request by epifrin**
Has written test that show bug explained in [issue 15](https://github.com/yiisoft/error-handler/issues/15)

### CODE REVIEW

**Comment by roxblnfk** [comment]
Thank, you helped detect that:

- We forgot to transfer tests for `HeaderHelper` to this repository
- `HeaderHelper` did not work correctly if the list of header values was passed simultaneously in strings and arrays

File: tests/ErrorCatcherTest.php [hunk 1]
```diff
@@ -108,15 +109,28 @@ public function testDefaultContentType(): void
...
+        $catcher = $this->getErrorCatcher($container);
+        $requestHandler = (new MockRequestHandler())->setHandleException(new \RuntimeException());
+        $response = $catcher->process(
+            new ServerRequest('GET', '/', ['Accept' => ['text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8']]),
+            $requestHandler
+        );
+        $response->getHeader(Header::CONTENT_TYPE);
+        $this->assertSame('text/html', $response->getHeaderLine(Header::CONTENT_TYPE));
+    }
+
```
Comment (roxblnfk) [inline tests/ErrorCatcherTest.php:124]:
```suggestion
```

**Review by roxblnfk — APPROVED** [review]
However, your PR remains useful with one amendment: you can delete the new test case, since it related to the `HtmlHelper`. And this test case i added in #18

**Comment by samdark** [comment]
@roxblnfk I've merged your fix. Would you please take part of tests that's needed? @epifrin please check if it's alright now.

**Comment by epifrin** [comment]
> @roxblnfk I've merged your fix. Would you please take part of tests that's needed? @epifrin please check if it's alright now.

Yes, it's alright now. Thank you!

**Pull request merged**